python/lsst/ctrl/pool/parallel.py

```diff
@@ -216,6 +216,22 @@ def submitCommand(self, scriptName):
 class SlurmBatch(Batch):
     """Batch submission with Slurm"""
 
+    @staticmethod
+    def formatWalltime(walltime):
+        """Format walltime as days-hours:minutes"""
```

Comment:
If not documenting in numpy docstring style, at least change to the following, so that people can see that walltime is in seconds without having to read the code:

"""Format walltime (in seconds) as days-hours:minutes"""